demo/js/demo.js

```diff
@@ -986,11 +986,14 @@ window.onload = function() {
 
         grid.setState(state);
 
-        behavior.setCellProperties(idx.HEIGHT, 16, {
+        // decorate height cell in row "17"
+        var rowOrdinal = 17;
+        behavior.setCellProperties(idx.HEIGHT, behavior.getHeaderRowCount() + (rowOrdinal - 1), {
```

Comment:
This calculation needed for now to accommodate the way cell props are now grid row addressed rather than data row addressed so that they can be used for all rows including headers and not just data. This nonsense will however fall away once we do the GRID-283 (Refactor vertical grid coordinate).